docs/how-to-guides/tracing/trace_span.md

```diff
@@ -0,0 +1,150 @@
+
+| Attribute | Type | Description | Examples | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) |
+|---|---|---|---|---|
+| framework | string | Identifies the framework within which the trace is captured.  | promptflow | `Required` |
+| node_name | string | Denotes the name of the flow node. | chat | `Conditionally Required` if the flow is a Directed Acyclic Graph (DAG) flow. |
+| span_type | string | Specifies the type of span, such as LLM or Flow. | LLM | `Required` |
+| line_run_id | string | Unique identifier for the execution run within PromptFlow. | d23159d5-cae0-4de6-a175-295c715ce251 | `Required` |
+| function | string | The function associated with the span. | search | `Recommended` |
+| session_id | string | Unique identifier for chat sessions. | 4ea1a462-7617-439f-a40c-12a8b93f51fb | `Opt-In` |
+| referenced.line_run_id | string | Represents the line run ID that is the source of the evaluation run. | f747f7b8-983c-4bf2-95db-0ec3e33d4fd1 | `Conditionally Required`: Only used in evaluation runs |
+| batch_run_id | string | The batch run ID when in batch mode. | 61daff70-80d5-4e79-a50b-11b38bb3d344 | `Conditionally Required`: Only used in batch runs |
+| referenced.batch_run_id | string | Notes the batch run ID against which an evaluation flow ran. | 851b32cb-545c-421d-8e51-0a3ea66f0075 | `Conditionally Required`: Only used in evaluation runs |
+| line_number | int | The line number within a batch run, starting from 0. | `1` | `Conditionally Required`: Only used in batch runs |
+| \_\_computed\_\_.cumulative_token_count.prompt | int | Cumulative token count of child nodes for prompts. [1] | `200` | `Recommended` |
+| \_\_computed\_\_.cumulative_token_count.completion | int | Cumulative token count of child nodes for completion responses. [1] | `80` | `Recommended` |
+| \_\_computed\_\_.cumulative_token_count.total | int | Total cumulative token count for both prompts and completions. [1] | `120` | `Recommended` |
+
+**[1]:** Cumulative token counts are propagated up the span hierarchy, ensuring each span reflects the total token count of all LLM executions within its scope.
+
+**Standard Events**
+
+In promptflow, events emitted by the PromptFlow framework follow the format below
+
+- event MUST has attributes
+- event attributes MUST contain a key named `payload`, which refers to the data carried within an event.
+- event attributes payload MUST be a JSON string that represent an object.
+
+| Event | Payload Description | Payload Examples | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) |
+|---|---|---|---|
+| promptflow.function.inputs | Input of a span | ```{"chat_history":[],"question":"What is ChatGPT?"}``` | `Required` |
+| promptflow.function.output | Output of span | ```{"answer":"ChatGPT is a conversational AI model developed by OpenAI."}``` | `Required` |
```

Comment:
In payload description field, should we say "Input of a function call" instead of "Input of a span"?

Reply:
Fixed